src/test/java/com/epam/tc/hw2/ex1/Exercise1.java

```diff
@@ -0,0 +1,156 @@
+package com.epam.tc.hw2.ex1;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import java.util.List;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Test;
+
+public class Exercise1 {
+    private WebDriver webDriver;
+
+    private WebElement webElement;
+
+    @BeforeTest
+    public void setUp() {
+        WebDriverManager.chromedriver().setup();
+    }
+
+    @AfterTest
+    public void clear() {
+        webDriver.quit();
+    }
+
+    @Test
+    public void firstExerciseTest() throws InterruptedException {
+        webDriver = new ChromeDriver();
+        webDriver.manage().window().maximize();
+
+        //        1. Open test site by URL
+
+        webDriver.navigate()
+                 .to("https://jdi-testing.github.io/jdi-light/index.html");
+
+        //        2. Assert Browser title
+
+        assertThat(webDriver.getTitle())
+            .as("Opening Home Page").isEqualTo("Home Page");
+
+        //        3. Perform log in
+
+        webElement = webDriver.findElement(By.id("user-icon"));
+        webElement.click();
+        webElement = webDriver.findElement(By.id("name"));
+        webElement.click();
+        webElement.sendKeys("Roman");
+        webElement = webDriver.findElement(By.id("password"));
+        webElement.click();
+        webElement.sendKeys("Jdi1234");
+        webElement = webDriver.findElement(By.id("login-button"));
+        webElement.click();
+
+        //        4. Assert Username is logged in
+
+        webElement = webDriver.findElement(By.xpath("//*[text()='Roman Iovlev']"));
+        String logNameResult = webElement.getText().toUpperCase();
+
+        assertThat(logNameResult)
+            .as("Username")
+            .isEqualTo("ROMAN IOVLEV");
+
+        //        5. Assert that there are 4 items on the header section are displayed
+        //        and they have proper texts
+
+        String[][] headerSectionArray = {
+            {"//*[text()='Home']", "HOME"},
+            {"//*[text()='Contact form']", "CONTACT FORM"},
+            {"//*[text()='Service']", "SERVICE"},
+            {"//*[text()='Metals & Colors']", "METALS & COLORS"}
+        };
+
+        for (String[] strings : headerSectionArray) {
+            webElement = webDriver.findElement(By.xpath(strings[0]));
+            String headerSectionText = webElement.getText().toUpperCase();
+            assertThat(headerSectionText)
+                .as("Text in the Header Section")
+                .isEqualTo(strings[1]);
+        }
```

Comment:
Please split it to 
1. css locator that returns all items
2. list with expected texts

convert list from first point above to list of actual texts
compare expected and actual lists using something like `assertThat(actual).....isEqualTo(expected)`

Reply:
done